### PR TITLE
[15.0][FIX] partner_statement: tests date relativity

### PR DESCRIPTION
**Pull request by chienandalu**
Depending on the day running the test, it will success or not.

@MiquelRForgeFlow I didn't go deep into the causes (I don't know the module), but it looks like the tests aren't resilient enough.

cc @Tecnativa

### CODE REVIEW

File: partner_statement/tests/test_activity_statement.py [hunk 1]
```diff
@@ -3,11 +3,14 @@
...
 from datetime import date
 
+from freezegun import freeze_time
+
 from odoo import fields
 from odoo.tests import new_test_user
 from odoo.tests.common import TransactionCase
 
 
+@freeze_time("2024-05-01", tick=True)
```
Comment (MiquelRForgeFlow) [inline partner_statement/tests/test_activity_statement.py:13]: Thanks, but I would put this in `test_onchange_aging_type` below instead. The problem comes with that 30 days comparison.

This  problem comes from
Selection_3350

Comment (pedrobaeza) [reply partner_statement/tests/test_activity_statement.py:13]: I don't see what you mean. Can you please propose the patch in other PR?

Comment (MiquelRForgeFlow) [reply partner_statement/tests/test_activity_statement.py:13]: Just freezing the specific test.

Selection_3351

**Review by pedrobaeza — APPROVED** [review]
Merging this one as one valid approach, although the test is partial for only months with 30 days.

/ocabot merge patch

**Pull request merged**